distribution_metadata.c

```diff
@@ -594,16 +543,11 @@ TupleToShardPlacement(HeapTuple heapTuple, TupleDesc tupleDescriptor)
 	ShardPlacement *shardPlacement = NULL;
 	bool isNull = false;
 
-	Datum idDatum = heap_getattr(heapTuple, ATTR_NUM_SHARD_PLACEMENT_ID,
```

Comment:
Put back `#define`s for queries and numbers where the query and number did not appear in the same function body.